StockBugFix/StockBugsPatches.cs

```diff
@@ -581,5 +582,58 @@ public static class SpaceHeater_States_InitializeStates_Patch {
 					PUtil.LogWarning("No SpaceHeater update handler found");
 			}
 		}
+
+		/// <summary>
+		/// Applied to FoodDiagnostic.CheckEnoughFood to fix the calories calculation in the diagnostics panel.
+		/// </summary>
+		[HarmonyPatch(typeof(FoodDiagnostic), "CheckEnoughFood")]
+		public static class FoodDiagnostic_CheckEnoughFood_Patch {
+			/// <summary>
+			/// Fix calories calculation.
+			/// </summary>
+			internal static bool Prefix(FoodDiagnostic __instance, ref ColonyDiagnostic.DiagnosticResult __result) {
+				__result = new ColonyDiagnostic.DiagnosticResult(ColonyDiagnostic.DiagnosticResult.Opinion.Normal, STRINGS.UI.COLONY_DIAGNOSTICS.GENERIC_CRITERIA_PASS);
+				if (__instance.tracker.GetDataTimeLength() < 10f) {
+					__result.opinion = ColonyDiagnostic.DiagnosticResult.Opinion.Normal;
+					__result.Message = STRINGS.UI.COLONY_DIAGNOSTICS.NO_DATA;
+				} else {
+					var dupes = Components.LiveMinionIdentities.GetWorldItems(__instance.worldID);
+					var trackerSampleCountSeconds = Traverse.Create<FoodDiagnostic>().Field("trackerSampleCountSeconds").GetValue<float>();
```

Comment:
This field should be accessible via a parameter named `___trackerSampleCountSeconds` to simplify the code

Reply:
oh, I was under the impression that this doesn't work with members of the base class. learned something :)